base/src/jsMain/kotlin/text/NumberFormatter.kt

```diff
@@ -20,28 +20,33 @@ package com.splendo.kaluga.base.text
 import com.splendo.kaluga.base.utils.Locale
 
 // TODO Implement with proper number formatter solution for Java Script
-actual class NumberFormatter actual constructor(actual val locale: Locale, style: NumberFormatStyle) {
-    actual var percentSymbol: Char = '%'
-    actual var perMillSymbol: Char = '\u2030'
-    actual var minusSign: Char = '-'
-    actual var exponentSymbol: String = "E"
-    actual var zeroSymbol: Char = '0'
-    actual var notANumberSymbol: String = "NaN"
-    actual var infinitySymbol: String = "\u221E"
-    actual var currencySymbol: String = ""
-    actual var currencyCode: String = ""
-    actual var positivePrefix: String = ""
-    actual var positiveSuffix: String = ""
-    actual var negativePrefix: String = "-"
-    actual var negativeSuffix: String = ""
-    actual var groupingSeparator: Char = ','
-    actual var usesGroupingSeparator: Boolean = false
-    actual var decimalSeparator: Char = '.'
-    actual var alwaysShowsDecimalSeparator: Boolean = false
-    actual var currencyDecimalSeparator: Char = '.'
-    actual var groupingSize: Int = 0
-    actual var multiplier: Int = 1
-
-    actual fun format(number: Number): String = "$number"
-    actual fun parse(string: String): Number? = string.toFloatOrNull()
+/**
+ * Default implementation of [BaseNumberFormatter]
+ * @param locale The [Locale] used for parsing. Defaults to [Locale.defaultLocale].
+ * @param style The [NumberFormatStyle] to configure the format to use. Defaults to [NumberFormatStyle.Decimal].
+ */
+actual class NumberFormatter actual constructor(override val locale: Locale, style: NumberFormatStyle) : BaseNumberFormatter {
+    override var percentSymbol: Char = '%'
+    override var perMillSymbol: Char = '\u2030'
+    override var minusSign: Char = '-'
+    override var exponentSymbol: String = "E"
+    override var zeroSymbol: Char = '0'
+    override var notANumberSymbol: String = "NaN"
+    override var infinitySymbol: String = "\u221E"
+    override var currencySymbol: String = ""
```

Comment:
Should we take it from the provided locale?

Reply:
The JS Locale doesnt contain any of these. Javascript need to be fully updated to account for all this, but its not possible for now